index.html

```diff
@@ -275,7 +266,49 @@ <h4 id="Checking_Your_Work">Checking Your Work</h4>
 					),
 				);
 			</pre>
-			
+
+			<h4 id="Checking_Your_Work">Checking Your Work</h4>
+
+			<div class="os-switch">
+				<div class="windows">
+					<p>In your browser, navigate to <a href="http://localhost/fuel_intro/public">http://localhost/fuel_intro/public</a> and you should see the &#8220;successful install&#8221; page below from Fuel.</p>
+				</div>
+				<div class="linux">
+					<p>
+							In order to run your php built-in server do the following:
+							<ol>
+								<li>Navigate to "fuel_intro/public"</li>
+								<li>Run the command "php -S localhost:8000"</li>
```

Comment:
Does this actually work?

I' thought it'd have to be run using the oil command to take into account all the fuelphp routing configuration.

http://fuelphp.com/docs/packages/oil/server.html